.changelog/19336.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+consul: Added support for failures_before_warning and failures_before_critical in Nomad agent services
+```
```

Comment:
This needs a second entry for the jobspec change (just a second `release-note:improvement` block under this one).

Reply:
Ah dang didn't realize how this worked -  I reran the "make cl" command twice thinking I could add 2 changelog entries, but I guess it overwrote the initial.